radiation/fortran/radlw/dump/Serialized_rank0_cdec.dat

```diff
@@ -0,0 +1 @@
+Y:%�"q�?
```

Comment:
why are you including serialized data in the repo? It looks like it is very small though, so perhaps it is ok to store here. but it is under a folder called 'dump', which makes me think perhaps it is not intentional to push this?

Reply:
strange, this wasn't intentional and this data has been here before now, I'll look into it

Reply:
I think serialized data has been removed now